handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -270,6 +281,9 @@ public String run() {
                 SSLContext.setOptions(ctx, SSL.SSL_OP_SINGLE_ECDH_USE);
                 SSLContext.setOptions(ctx, SSL.SSL_OP_SINGLE_DH_USE);
                 SSLContext.setOptions(ctx, SSL.SSL_OP_NO_SESSION_RESUMPTION_ON_RENEGOTIATION);
+                // We not support compression as the moment so not use it.
+                SSLContext.setOptions(ctx, SSL.SSL_OP_NO_COMPRESSION);
```

Comment:
yeah it was never supported... We just not disabled it explicit before which we should do anyway.